module.json

```diff
@@ -1,12 +1,16 @@
 {
     "main": {
         "name": "ct.nakama",
-        "tagline": "Implementation of heroiclabs/nakama to enable multiplayer games",
-        "version": "0.0.2",
+        "tagline": "A wrapper for nakama-js to enable multiplayer games",
+        "version": "2.3.0",

```

Comment:
Why `2.3.0`? I would say bump it up to `0.0.3`

Reply:
the idea is to match the version number of nakama-js. (just a thought)